data/mods/ssb/random-teams.js

```diff
@@ -168,6 +174,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Loving Embrace',
 				evs: {hp: 4, spa: 252, spe: 252}, nature: 'Modest', shiny: true,
 			},
+			'DaWoblefet': {
+				species: 'Wobbuffet', ability: 'Shadow Artifice', item: 'Iapapa Berry', gender: 'M',
+				moves: ['Counter', 'Mirror Coat', 'Encore'],
+				signatureMove: 'Super Ego Inflation',
+				evs: {spa: 252, spd: 4, spe: 252}, ivs: {spe: 0}, nature: 'Relaxed',
```

Comment:
Unless this was a nerf, the EVs are wrong here - should be 252 HP / 252 Def / 4 Sp. Def,

Reply:
Nope, just an oversight, thanks for letting me know :)